grimoirelab/toolkit/datetime.py

```diff
@@ -67,7 +67,8 @@ def datetime_to_utc(ts):
 
     Returns the given datetime object converted to a date with
     UTC+0 timezone. For naive datetimes, it will be assumed that
-    they are in UTC+0.
+    they are in UTC+0. When the timezone is not wrong, UTC+0 will
```

Comment:
I guess you mean `when the timezone is wrong`. Or better:

```
Returns the given datetime object converted to a date with
UTC+0 timezone. When the timezone is wrong, the date will
be converted removing the timezone info. For naive datetimes,
it will be assumed they are in UTC+0
```